src/lib/Metadata/Reduce.cpp

```diff
@@ -298,6 +299,28 @@ void merge(FriendInfo& I, FriendInfo&& Other)
         I.FriendType = std::move(Other.FriendType);
 }
 
+void merge(AliasInfo& I, AliasInfo&& Other)
+{
+    MRDOCS_ASSERT(canMerge(I, Other));
+    if (I.AliasedSymbol != nullptr)
+        I.AliasedSymbol = std::move(Other.AliasedSymbol);
+    mergeSourceInfo(I, std::move(Other));
+    mergeInfo(I, std::move(Other));
+}
+
+void merge(UsingInfo& I, UsingInfo&& Other)
+{
+    MRDOCS_ASSERT(canMerge(I, Other));
+
+    reduceSymbolIDs(I.UsingSymbols, std::move(Other.UsingSymbols));
+    if (I.Class != UsingClass::Normal)
```

Comment:
This condition needs to be inverted. We only want to overwrite `Class` if it has its default value.